spec/teacher_spec.rb

```diff
@@ -0,0 +1,17 @@
+require_relative './spec_helper'
+
+describe Teacher do
+  teacher = Teacher.new('Math', 25, name: 'Camila')
```

Comment:
- [ ] Please respect the way arguments are ordered in the Teacher class. Take a look at the following picture 👇 
![image](https://user-images.githubusercontent.com/90222110/212169628-6821fdf7-f00d-4758-af34-998e52fd3901.png)
AS you can see, age comes first and the specification follows 😉

- [ ] Also the third argument is supposed to be the name, therefore you don't have to mention the argument name (i.e: the `name:` you added )
I suggest you do it in that way
`teacher = Teacher.new(25, 'Math', 'Camila')`